### ISSUE TITLE
About the docker image

### ISSUE THREAD

**Issue by dlfming** [shinke-li/Campus3D#11]
i can't successfully run the command “docker build -t shinkeli/campus3d:latest .” , and i even can't find the image "shinkeli/campus3d" in the Docker Hub, but i find one named "shinkeli/conda-tf2-open3d", which published before 4 years ago, is this the right image for the project?

**Comment by shinke-li** [comment]
Yes, you may attempt to use this image. However, as docker is no longer supported in some countries, it is hard to keep it updated. For the sake of convenient env building, I recommend you use the reproduce version in https://github.com/Yuqing-Liao/reproduce-campus3d, which is based on pytorch.
